exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -589,8 +589,147 @@ func (p *Poloniex) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse, er
 
 // GetOrderInfo returns order information based on order ID
 func (p *Poloniex) GetOrderInfo(orderID string, pair currency.Pair, assetType asset.Item) (order.Detail, error) {
-	var orderDetail order.Detail
-	return orderDetail, common.ErrNotYetImplemented
+	var orderInfo order.Detail
+	trades, err := p.GetAuthenticatedOrderTrades(orderID)
```

Comment:
I think it's better to return an error.  We can always range over trade returns that could be nil. This reduces another potential outbound call from p.GetAuthenticatedOrderStatus.

Reply:
Sorry, I don't quite understand your comment. I am trying to get complete information about the order, including trades (if any). On the closed orders we will get info from the GetAuthenticatedOrderTrades method, and from the GetAuthenticatedOrderStatus for open orders. And I think on partially filled orders we will get info from both methods.

Concerning "Is there a specific reason you have done the checks above this? I would just revert back to result.Date.Unix() and closetime." - reason why I make the checks that if we get no Date from API responses, we will get negative Timestamp after .Unix() conversion

Reply:
maybe something like this:

```go
trades, err := p.GetAuthenticatedOrderTrades(orderID)
	if err != nil && err.Error() != "Order not found, or you are not the person who placed it." {
		return order.Detail{}, err
	}
```

So if there is an api key issue or malformed orderNumber we don't send another request. Let me know what you think.

Reply:
Thanks for clarifying on the Unix output, I will resolve.